src/doc/guide.md

```diff
@@ -419,6 +419,17 @@ will not fail your overall build. Please see the [Travis CI Rust
 documentation](https://docs.travis-ci.com/user/languages/rust/) for more
 information.
 
+
+## Build cache
+
+Cargo shares build artifacts among all the packages of a single workspaces.
+If you want to share build results across different workspaces, you can use
+[sccache]. For the minimal setup, install sccache with `cargo install sccache`
+and add `export RUSTC_WRAPPER=sccache` to the shell init script.
```

Comment:
`export RUSTC_WRAPPER=sccache` is horribly Unix specific, but I don't know a better way of saying: "Hey, you need to set up this env var to this value, and you must do it on every logic, it's not a one-time thing".

Reply:
Perhaps this could mention the environment variable and then say something like "for example if you're using a bash shell you can do..."